Bug59294_BetterPercentileLabelv3

ra0077 opened the pull request:
Hi,

I propose better percentile label :
   new and better label
   same in Response Time Per Sample and Dashboard statistics

New release with the right properties files
Antonio

ra0077 committed:
Bug59294_BetterPercentileLabelv3

Hi,

I propose better percentile label :
   new and better label
   same in Response Time Per Sample and Dashboard statistics

New release with the right properties files
Antonio

milamberspace commented on src/core/org/apache/jmeter/resources/messages_fr.properties:
```diff
@@ -853,7 +853,7 @@ reportgenerator_summary_statistics_kbytes=Ko/sec
 reportgenerator_summary_statistics_max=Max
 reportgenerator_summary_statistics_min=Min
-reportgenerator_summary_statistics_percentile_fmt=%d%% Ligne
+reportgenerator_summary_statistics_percentile_fmt=%de percentile
```
In French, percentile is translate by the word "centile"
https://fr.wikipedia.org/wiki/Centile

For example : "99e centile"

ra0077 replied:
Hi Milamber,

I am ok with your but in wikipedia (https://fr.wikipedia.org/wiki/Centile)
I have seen that percentile is also accepted

I quoted wikipedia:
"En statistique descriptive
<https://fr.wikipedia.org/wiki/Statistique_descriptive>, un *centile* est
chacune des 99 valeurs qui divisent les données triées en 100 parts égales,
de sorte que chaque partie représente 1/100 de l'échantillon de population.
(La dénomination anglaise <https://fr.wikipedia.org/wiki/Anglais> —
*percentile*, avec une prononciation francisée, est parfois utilisée)."

It's why I have prefered percentile instead of centile

But if you prefer centile, it's ok for me too

Antonio


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=oa-2200-c>
Garanti
sans virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=oa-2200-c>
<#DDB4FAA8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2016-04-09 9:23 GMT+02:00 Milamber <notifications@github.com>:

> In src/core/org/apache/jmeter/resources/messages_fr.properties
> <https://github.com/apache/jmeter/pull/187#discussion_r59111727>:
>
> > @@ -853,7 +853,7 @@ reportgenerator_summary_statistics_kbytes=Ko/sec
> >  reportgenerator_summary_statistics_label=Libell\u00E9
> >  reportgenerator_summary_statistics_max=Max
> >  reportgenerator_summary_statistics_min=Min
> > -reportgenerator_summary_statistics_percentile_fmt=%d%% Ligne
> > +reportgenerator_summary_statistics_percentile_fmt=%de percentile
>
> In French, percentile is translate by the word "centile"
> https://fr.wikipedia.org/wiki/Centile
>
> For example : "99e centile"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/jmeter/pull/187/files/0165352fa09a2771b19290c72f5425a875cf1513#r59111727>
>

milamberspace replied:
In JMeter (aggregate graph, aggregate report), we use "centile", so it's will better to continue to use the word.

ra0077 replied:
Ok no problem
I will compare aggregate graph/aggregate report with HTML to use the same words

Antonio

ra0077 replied:
Hi Milamber,

I have compare HTML report with other listener and find another difference (in % error)

I have commit it in https://github.com/apache/jmeter/pull/190 because percentil PR (PR 187) modify also a java file and not only properties files

Antonio

vlsi commented:
@ra0077, thanks for the PRs, however could you please try reusing the PRs next time?

Regarding "the right properties files", can you please clarify if you have to perform additional steps to make properties file "sane"?


Regarding reuse:

You could force push new commit to your `Bug59294_BetterPercentileLabel` branch, that would update #186.

In simple case (like #186) "close PR + send new PR" does little difference from "update PR with new contents", however keeping the same URL, comment thread, conversation, etc, helps.

ra0077 commented:
Hi Vladimir,

Thanks to your tips and advices

I have juste make a rebase to have "clean properties files" and I have
modify again the 2 files

Antonio

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=oa-2200-c>
Garanti
sans virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=oa-2200-c>
<#DDB4FAA8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2016-04-09 9:32 GMT+02:00 Vladimir Sitnikov <notifications@github.com>:

> @ra0077 <https://github.com/ra0077>, thanks for the PRs, however could
> you please try reusing the PRs next time?
>
> Regarding "the right properties files", can you please clarify if you have
> to perform additional steps to make properties file "sane"?
>
> Regarding reuse:
>
> You could force push new commit to your Bug59294_BetterPercentileLabel
> branch, that would update #186 <https://github.com/apache/jmeter/pull/186>
> .
>
> In simple case (like #186 <https://github.com/apache/jmeter/pull/186>)
> "close PR + send new PR" does little difference from "update PR with new
> contents", however keeping the same URL, comment thread, conversation, etc,
> helps.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/jmeter/pull/187#issuecomment-207733923>
>

ra0077 committed:
Bug59294_BetterPercentileLabelv3 

Modify French label from percentile to centile to be like in another
listener

Antonio
